Tests/UnitTests/Reporters/JunitXMLReporterTest.jsl

```diff
@@ -148,12 +158,12 @@ ut test( JunitXMLReporters, "JunitXMLReporter one simple passing", Expr(
 	     ut assert that(Expr(1+1), ut equal to (2));
 	   ));
 	));
-	ut assert that(Expr(reporter << successes), ut equal to ({{"Simple Pass ⮚ One plus one equals two ⮚ 1", 1 + 1, "equal to 2", Empty()}}));
+	ut assert that(Expr(reporter << n successes), ut equal to (1));
 	pattern = 
 	"\[<\?xml version="1.0" encoding="UTF-8" \?>
-	<testsuites failures="0" id="\d+_\d+" tests="1">
-	  <testsuite failures="0" name="Simple Pass" tests="1">
-		<testcase name="One plus one equals two">
+	<testsuites errors="0" failures="0" id="\d+_\d+" tests="1">
+	  <testsuite errors="0" failures="0" name="Simple Pass" tests="1">
+		<testcase classname="Simple Pass" file="JunitXMLReporterTest\.jsl" name="One plus one equals two">
```

Comment:
Maybe a special case for when `Include File List[1]` is `""`?

Reply:
Just curious: is there a case where that can happen?

Reply:
Including a blob definitely does it. See `Include(Char To Blob(JSL Quote(Include File List())));`

Reply:
I was assuming that the `\.jsl` in this test was because `Include File List()` had a `""` in there during the running of the actual test. I may be wrong about that. Do the tests still pass after your special case change in 56aaf52?

Reply:
Oh the `\.` of `\.jsl` is because this is a regex pattern that we're matching to. I wanted to escape the `.` in the pattern so the test is more correct. I think we can't match the string directly because the id changes from run to run? 

Yes the tests still pass. Interesting about including a blob - thanks for the example.